projects/batfish-common-protocol/src/test/java/org/batfish/datamodel/pojo/AnalysisTest.java

```diff
@@ -0,0 +1,90 @@
+package org.batfish.datamodel.pojo;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import com.google.common.testing.EqualsTester;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import org.batfish.common.BatfishException;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Tests for {@link Analysis}. */
+@RunWith(JUnit4.class)
+public class AnalysisTest {
+
+  @Rule public ExpectedException _thrown = ExpectedException.none();
+
+  @Test
+  public void testConstructorWithName() {
+    Analysis a = new Analysis("analysis");
+    assertThat(a.getName(), equalTo("analysis"));
+    assertTrue(a.getQuestions().isEmpty());
+  }
+
+  @Test
+  public void testBasicFunctions() {
+    Map<String, String> questions = Collections.singletonMap("question", "questionContent");
+    Analysis a = new Analysis("analysis", questions);
+    assertThat(a.getName(), equalTo("analysis"));
+    assertThat(a.getQuestions(), equalTo(questions));
+    a.setName("other-analysis");
```

Comment:
these tests seem very redundant. Why do you test that the constructor worked here, you already tested it?